### DISCUSSION TITLE
branding/copyright

### DISCUSSION THREAD

**Discussion by fragolinux** [dani-garcia/vaultwarden#1619] [General]
hi, we'd like to implement (moving from a passbolt installation) a cloud served bitwarden_rs installation, and we're going to have a security review in next weeks.

Our concerns are about the "bitwarden inc" copyright signatures in the vault footer... is this allowed even if _rs is an unofficial but api compatible (re)build? Or is [recompile the vault](https://github.com/dani-garcia/bitwarden_rs/wiki/Building-binary#install-the-web-vault) image a better/safer solution?

It does not seem to be possible to get rid of those copyrights using the [template system](https://github.com/dani-garcia/bitwarden_rs/tree/master/src/static/templates/), and honestly i don't know if removing the branding is even more illegal than leaving it there on an unofficial derived product.... any hint? Thanks in advance

**Comment by fragolinux** [comment]
hi, any suggestion or comment, please? Thanks

**Comment by dani-garcia** [comment]
@fragolinux I've been in talks with Kyle (Founder of bitwarden inc) about rebranding the project to avoid confusion, and he didn't have any issue with the footer copyright itself, we've decided to change the footer to add a "powered by unofficial server" but mostly to make it easier for support issues.

With that said I'm not a lawyer, but seeing as the web vault we distribute is pretty much unmodified, I think it should be fine to keep the original copyright footer as it is. Don't think you can remove them though, that's up to the GPLv3 license, and I'm pretty sure that's not allowed.

**Reply by pepa65** [reply]
>  Don't think you can remove them though, that's up to the GPLv3 license, and I'm pretty sure that's not allowed.

According to the GPLv3 you can distribute the the software and make any changes, as long as you include the "copyright notice" (`LICENCE.txt`). You don't have to advertise any copyright anywhere else, so I am pretty sure the GPLv3 allows changing or removing it.
